route_info/src/lib.rs

```diff
@@ -105,6 +105,7 @@ impl RouteInfo {
                                 road.reference_line
                                     .to_geojson(Some(&self.network.gps_bounds)),
                             );
+                            feature.id = Some(Id::Number((1 + features.len()).into()));
```

Comment:
The new hover/tooltip component needs feature IDs, just filling these in for the speed limit layer